---
title: "The diversity and functional capacity of microbes associated with coastal phototrophs"
collection: publications
permalink: /publication/2022-01-10-coastal-microbes
excerpt: 'Coastal marine phototrophs exhibit some of the highest rates of primary productivity in the world. They have been found to host a diverse set of microbes, many of which may impact the biology of their phototroph hosts through metabolisms that are unique to microbial taxa. Here we characterized the metabolic functions of phototroph-associated microbial communities using metagenomes collected from 2 species of kelp (Laminaria setchellii and Nereocystis luetkeana) and 3 marine angiosperms (Phyllospadix scouleri, P. serrulatus and Zostera marina), including the rhizomes of two surfgrass species (Phyllospadix spp.) and the seagrass Zostera marina, and the sediments surrounding P. scouleri and Z. marina. Using metagenomic sequencing, we describe 72 metagenome assembled genomes (MAGs) that potentially benefit from being associated with macrophytes and may contribute to macrophyte fitness through their metabolic gene content. All host-associated metagenomes contained genes for the use of dissolved organic matter from hosts and vitamin (B1, B2, B7, B12) biosynthesis. Additionally, we found a range of nitrogen metabolism genes that transform dissolved inorganic nitrogen into forms that may be more available to the host. The rhizosphere of surfgrass and seagrass contained genes for anaerobic microbial metabolisms, including nifH genes associated with nitrogen fixation, despite residing in a well-mixed and oxygenated environment. The range of oxygen environments engineered by macrophytes likely explains the diversity of both oxidizing and reducing microbial metabolisms, and contributes to the functional capabilities of microbes and their influence on carbon and nitrogen cycling in nearshore ecosystems.'
date: 2022-01-10
venue: 'Submitted to mSystems'
paperurl: 'https://www.biorxiv.org/content/10.1101/2022.01.05.475171v1.full.pdf'
citation: 'Miranda, K.K., Weigel, B.L., Fogarty, E.C., Veseli, I.A., Giblin, A.E., Eren, M.A. and Pfister, C.A. (2022), The diversity and functional capacity of microbes associated with coastal phototrophs. mSystems <i>in Review</i>'
---
The microbial world is the new frontier for scientists as we are finally equipped with the tools to not only visualise the microsopic universe but also to detect what all these billions of microbes are up to. These microbial communities are everywhere, from your gut to hydrothermal vents sitting at the bottom of the ocean. But these microbes may not be simple spectators as previously thought and might in fact have a large impact on the food web. This was what we were trying to understand through this study - how may microbes living on kelp and surfgrasses interact with their hosts? These hosts that we've chosen are crucial for the nearshore food web as they form the base for all food available as well as the structure for other animals in higher trophic levels. Furthermore, kelp forests are some of the most productive ecosystems in the <i>WORLD</i>. Could it be the little guys enabling this staggering productivity? 

<p align="center">
<img src="/images/tatoosh_forest.JPG">
<figcaption align="right"><i>The unimaginable productivity of the underwater forests of Tatoosh Island </i></figcaption>
</p>

To study these microbial communities, we sampled <i>Laminaria setchellii</i> and <i>Nereocystis luetkeana</i> (kelp), <i>Phyllospadix spp.</i> (surfgrasses) and <i>Zostera marina</i> (a seagrass from the Atlantic coast). The microbial communities had never been described for these kelps and surfgrasses so we're in completely new territory here! Because of the new frontiers we were microscopically discovering, we included the well studied <i>Zostera</i> microbial community to confirm any new findings that we make. Of the gazillion things microbes do, we were curious in identifying the cycling of nitrogan and carbon - microbial currency. Specifically, we were on the lookout for nitrogen fixation - the process responsible for making all our legumes so rich in proteins! The process in itself is energy intensive and is usually found in regions that are really lacking in nitrogen, however, modern genetic techniques are finding this even in regions of the world that have a lot of nitrogen, like Tatoosh Island which sits on top of an upwelled current bringing in nutrient rich water from the deep Pacific Ocean. If kelps and surfgrasses have this metabolism in their back pocket, this could explain it all.

This project showed me how rapidly advanced our microbial techniques have become. When my mother did her PhD., she studied a single process, denitrification, in E. Coli (taking ammonium and converting it to nitrogen gas - the reverse process that I'm currently studying! THE CYCLE IS COMPLETE). Now, we have the tools to research <i>every</i> process going on in <i>communities</i> of microbes. The first step is we extract all the DNA from a tiny tissue sample from kelps or surfgrass no bigger than my pinky nail (an above average pinky). But the DNA is massive in size, impossible to read directly. So we employ a process called shotgun metagenomic sequencing followed by genetic reconstruction using fancy computer software. Think about this in terms of assembling a puzzle you've never seen before. We shake the box with the fully formed puzzle inside to break up the pieces (shotgun metagenomic sequencing) and then reconstruct it 

<p align="center">
<img src="/images/rhizosphere.JPG">
<figcaption align="right"><i>Stinky rhizomes of surfgrass. The tiniest little attachments keep massive meadows glued onto sheer rock! </i></figcaption>
</p>

So what did we actually find?! Besides describing novel microbial communities for the first time, we found that access to oxygen played a massive role in structuring who was present in our communities and what they could do. Kelp and surfgrass blades are bobbing around in this wave beaten environment that constantly cycling nutrients and oxygen in the water. However, the rhizosphere (the mud and root-like tissue that keeps surfgrasses attached to the rocks) have almost no oxygen present. It's this tightly packed network of root tissue and mud that releases a sharp shaft of ammonium up your nostrils to moment you lift it up. I used a teeny tiny oxygen sensor and found almost no oxygen present compared to the surrounding seawater which is shockingly usique to this otherwise well oxygenated environment. Oxygen is good currency in the microbial world but when microbes get oxygen-strapped, they have to start getting creative! They can use sulfate or nitrate to do the same job as oxygen but they're far more innefficient. As wonderful as oxygen is, it's also so energetic that it is damaging. So when oxygen is away, microbes can interact with nitrogen and carbon in unique and diverse ways that allow for more efficient recycling. Indeed, the diversity of nitrogen processes was far more greater in low oxygen environments where we also found microbes commonly associated with sulfur cycling. We also found evidence for nitrogen fixation! Like expected, we found n-fixers in <i>Zostera</i> and for the first time, in the rhizome of <i>Phyllospadix</i>.

If you want to get into the nitty gritties of the paper, check it out [here](https://www.biorxiv.org/content/10.1101/2022.01.05.475171v1.full.pdf)

<p align="center">
<img src="/images/o2_testing.JPG">
<figcaption align="right"><i>Sampling the oxygen environment of surfgrass!</i></figcaption>
</p>

Recommended citation: Miranda, K.K., Weigel, B.L., Fogarty, E.C., Veseli, I.A., Giblin, A.E., Eren, M.A. and Pfister, C.A. (2022), The diversity and functional capacity of microbes associated with coastal phototrophs. mSystems <i>in Review</i>.

<a href="https://github.com/kkmiranda/PNWMetagenomes">GitHub Link</a>

